src/qt/transactiondesc.cpp

```diff
@@ -57,25 +55,9 @@ QString TransactionDesc::FormatTxStatus(const CWalletTx& wtx)
             }
         }
 
-        if (wtx.IsLockedByLLMQInstantSend()) {
+        if (wtx.IsLockedByInstantSend()) {
             strTxStatus += " (" + tr("verified via LLMQ based InstantSend") + ")";
```

Comment:
I'm not sure tacking on LLMQ to these features is good from a usability perspective. LLMQ looks crazy, InstantSend anyone can understand.
( This had another comment thread but was marked resolved and outdated so bringing it up here )

Reply:
I answered in the original thread

Reply:
Your initial idea was to remove the "via LLMQ" udjin thought we should keep it so you reverted that change and I'm saying we should get rid of it as you originally suggested since "LLMQ" isn't consumer friendly

Reply:
Did you read my answer in the original thread? (https://github.com/dashpay/dash/pull/3020#discussion_r301221967)

I generally agree that we should get rid of it, but I also agree to what udjin says as it might confuse people when we do it in the next release.

Reply:
I agree as well btw :) Let's keep this out of the scope of this specific PR and remove "LLMQ" part from both IS and ChainLocks strings in some another PR in the future.

Reply:
Okay, agreed for later

Reply:
Sorry about that @codablock, for some reason either I missed your comment or github wasn't showing it to me, because I looked for a response just before I made this comment thread

Reply:
Don't worry, happened to me as well already. Github seems to get confused with comments/threads when force-pushes are involved. Best is to enable email notifications for comments/mentions, this way you never miss a comment that is happening in one of the threads (and PRs) where you're involved.